assets/in

```diff
@@ -8,13 +8,14 @@ set -o nounset
 exec 3>&1 # make stdout available as fd 3 for the result
 exec 1>&2 # redirect all output to stderr for logging
 
+pip install -r /opt/resource/requirements.txt
```

Comment:
I think this pip install is only installing requirements that were not needed anyway so can be removed.